external/Eigen/src/IterativeLinearSolvers/IncompleteLUT.h

```diff
@@ -95,7 +95,7 @@ Index QuickSplit(VectorV &row, VectorI &ind, Index ncut)
   * alternatively, on GMANE:
   *   http://comments.gmane.org/gmane.comp.lib.eigen/3302
   */
-template <typename _Scalar, typename _StorageIndex = int>
```

Comment:
Please try not to modify eigen source code. Just go to the place where Eigen::IncompleteLUT<..> is initialized in G+Smo source code and put there the template argument that is needed.